src/api/app/controllers/webui/cloud/ec2/configurations_controller.rb

```diff
@@ -7,6 +7,8 @@ class ConfigurationsController < WebuiController
         before_action -> { feature_active?(:cloud_upload) }
 
         def show
+          # FIXME: remove it when cloud upload are ready to production
+          switch_to_webui2 if Rails.env.development?
```

Comment:
For this action, the view is already fully migrated in this PR. We don't need to check for the environment.

Reply:
ah true, just in the monitor pages, since it's still under construction

Reply:
Check removed.